index.html

```diff
@@ -483,19 +483,19 @@ <h5 class="card-title">Inside the Mysterious Area 51: Aliens, UFOs, Conspiracies
 
 
         <!-- IN THE MAGAZINE SECTION -->
-        <div class="magazine-section">
-          <div class="d-flex justify-content-center border-bottom">
+        <div class="magazine-section col-sm-12 col-md-12 my-3 bg-white mx-auto">
+          <div class="d-flex justify-content-center border-bottom py-4">
             <div class="d-flex align-items-center">
               <div class="n-icon"></div>
               <div class="sign-up-box-title pl-4 text-uppercase">in the magazine</div>
             </div>
           </div>
 
-          <div class="row">
-            <div class="magazine-card my-2 col-lg-3 col-md-3 col-sm-12">
+          <div class="row mt-4">
+            <div class="magazine-card my-2 col-lg-3 col-md-3 col-sm-12 align-items-baseline">
               <div class="card rounded-0 border-0 position-relative">
                 <div class="position-relative">
-                  <img src="/image/magazine-img-1.webp" class="card-img-top rounded-0" alt="tech academy">
+                  <img src="/image/magazine-img-1.jpg" class="card-img-top rounded-0" alt="tech academy">
                   <a href="#" class="category text-uppercase">july 17 issue</a>
                 </div>
               </div>
```

Comment:
**[Use semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp)**

You still have to use `<section>` and `<article>` tags, use divs where is necessary.